### PR TITLE
Release/1.0.5

### PR DESCRIPTION
**Pull request by kylebaron**

### CODE REVIEW

**Comment by kylebaron** [comment]
# macbuilder (M1)

```
* using log directory ‘/Volumes/PkgBuild/work/1660836376-4d45a9f6cb369569/packages/big-sur-arm64/results/4.2/mrgsolve.Rcheck’
* using R version 4.2.1 Patched (2022-06-23 r82516)
* using platform: aarch64-apple-darwin20 (64-bit)
* using session charset: UTF-8
* checking for file ‘mrgsolve/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘mrgsolve’ version ‘1.0.4.9000’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for executable files ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘mrgsolve’ can be installed ... [23s/23s] OK
* checking installed package size ... NOTE
  installed size is  7.6Mb
  sub-directories of 1Mb or more:
    libs   5.8Mb
* checking package directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... [4s/4s] OK
* checking Rd files ... [0s/0s] OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... OK
* checking examples ... [3s/3s] OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ... [5s/6s] OK
  Running ‘testthat.R’ [5s/5s]
* checking PDF version of manual ... OK
* DONE
Status: 1 NOTE
* using check arguments '--no-clean-on-error '

* elapsed time (check, wall clock): 0:54
* ```

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+# mrgsolve 1.0.5
+
+- Changed behavior for dosing records where EVID = 4 and SS != 0 
+  to match what is done by NONMEM: the system will be advanced to 
+  steady-state but will not be reset; behavior prior to 1.0.5 
+  release was to advance to steady-state and then reset (#1011).
+
+- Any column in an input data set that has a class attribute will be
```
Comment (kyleam) [inline NEWS.md:8]: typo: extra "be"

**Review by kyleam — APPROVED** [review]
Release-specific bits look good to me, aside from NEWS typo.

File: NEWS.md (hunk 1 above)
Comment (kylebaron) [reply NEWS.md]: Thanks.

**Pull request merged**